unix/README.md

```diff
@@ -26,7 +26,11 @@ following packages pre-installed:
     libjpeg8-dev
     libtiff5-dev
     libopenexr-dev
-    
+
+To enable the, as rendering, Simple DirectMedia Layer display also pre-install:
```

Comment:
Good point. So maybe something like: 

"To enable the render preview display also pre-install:" 

No, we don't support an X-Windows display. SDL though still uses X11 for a small subset of functionality and so the X11 library is a dependency of SDL (-lX11 -lXext). In the unlikely case X11 isn't already present on some unix system - the installation of the SDL library will cause all the X11 dependencies to be installed. I'll add a note something like "the --with-x configure option is currently meaningless as no X11 display option is presently implemented."

If we want to go further with updates we should probably strike the text: 

"NB if you have improvements to the below instructions, please add a comment
to issue #1, or (if you are working in a branch) send us a pull request."

as this looks like some leftover text from years past to me. 

I also think we should not be specifying versions for the following development libraries. Today we have:

zlib1g-dev 
libpng12-dev
libjpeg8-dev
libtiff5-dev

where I think we should have: 

zlib-dev 
libpng-dev
libjpeg-dev
libtiff-dev

Let ./configure determine whether the default versions are OK. In other words, by specifying the library version we are encouraging people to install those versions specifically and they both might be old, and they probably won't be picked up where old. 

For example, our png m4 macro today looks for a sequence of library versions in the following order: "png14 png png12 png" using our AX_CHECK_LIB macro. First noting that debian based linuxes are currently planning to jump from png12 to png16 post "Ubuntu 16.04" as "the" png library, folks are going to get png16 on those newer Debians no matter installing png12. In standard set ups libpng will be sym linked to libpng16 for the sanity of the overall install. Drifting deeper, this gets to a ./configure issue in our set up that though systems can have multiple versions of libraries installed in the same standard directories, only one of those is the valid system lib<basename> default. If we want to enable use of other than default standard libaries longer term, I think what would might be good is to extend or modify our "--with-libpng[=DIR]" support such so it is something more like: "--with-libpng[=DIR | specifc library version]". 

I think adding the the following note a good idea too given what you found with OSX and pkg-config: 

Lastly, certain OSX environments do not install the program pkg-config
with autoconf.  The program is currently necessary when 
determining the link library flags for openexr.  In these situations
also install:


pkg-config

 

I've made all the changes discussed above and will commit and push and update to this pull request shortly. We can further refined the changes if need be.